fmmax/fmm.py

```diff
@@ -28,6 +28,10 @@ class Formulation(enum.Enum):
     JONES: str = vector.JONES
     NORMAL: str = vector.NORMAL
     POL: str = vector.POL
+    JONES_DIRECT_FOURIER: str = vector.JONES_DIRECT_FOURIER
+    JONES_FOURIER: str = vector.JONES_FOURIER
+    NORMAL_FOURIER: str = vector.NORMAL_FOURIER
+    POL_FOURIER: str = vector.POL_FOURIER
```

Comment:
Would be nice to describe each of these in the docstring, along with an overall recommendation. That way these details appear in the online docs.